docs/vgclog.md

```diff
@@ -0,0 +1,423 @@
+
+A garbage collection event is triggered in the following situations: 
+
+- The VM calls \system.gc() to run a global *stop-the-world* GC. 
+
+- A memory allocation failure. Allocation failures are triggered when a request to allocate an object fails due to a full region of the heap. 
+
+- Available memory in a region of the heap size decreased to a threshold value that triggers a preemptive cleaning of the heap.
+
+- Intermediate or final stages of a concurrent GC cycle are reached. A *stop-the-world* collection is run to complete the incremental or final stage. 
+
+Verbose GC logs contain information on garbage collection operations to assist with the following actions:  
+
+- Tuning GC performance to reduce pause times and speed up GC operations
+
+- Troubleshooting GC operations and policies. For example, analyzing long pauses, or determining how free memory is divided in the heap before and after a GC cycle. 
+
+Verbose GC logs, when enabled, begin capturing information as soon as garbage collection is initialized. By default, the information is printed to STDERR. The information can also be [printed to a file](vgclog.md#how-to-generate-a-verbonse-garbage-collection-log).
+
+To help you visualize and analyze the garbage collection, you can feed verbose GC log files into various diagnostic tools and interfaces. Examples include tools such as the [Garbage Collection and memory visualizer extension for Eclipse](https://marketplace.eclipse.org/content/ibm-monitoring-and-diagnostic-tools-garbage-collection-and-memory-visualizer-gcmv), and online services such as [GCEasy](https://gceasy.io). 
+
+**Note:** You can run one or more GC traces by using the (['-XTgc' option](xtgc.md)) to get more detailed information to help diagnose GC problems or perform finer tuning.
+
+## How to generate a verbose garbage collection log 
+
+You can enable verbose GC logs by specifying the `-verbose:gc` standard option when you start your application. For more information, see [standard command-line options](cmdline_general.md).
+
+The output of `-verbose:gc` is printed to STDERR by default. To print the log output to a file, append the `-Xverbosegclog` option. You can print the verbose GC log file to a succession of files, where each file logs a specified number of GC cycles, by using additional parameters of `-Xverbosegclog` option. For more information, see the command-line option [-Xverbosegclog](xverbosegclog.md).
+
+## Verbose GC log contents 
+
+The Verbose garbage collection logs are printed in XML format and consist of the following sections:  
+
+- a summary of your GC configuration, which is captured in the stanza beginning with the [`<INITIATIZED>` tag](vgclog.md#xml-tags-an-attributes) 
```

Comment:
The tag looks to be called `<initialized>` (lower case), which makes more sense. 

That section looks to contain not just a summary of the GC configuration, but system information, and the VM options in force for the application. I guess you plan to add an example section in the  gc_vgclog_tags.md file?

Reply:
Yep :) This is currently being developed on a separate PR.